waypoints/Alps-mountains.cup

```diff
@@ -0,0 +1,3884 @@
+name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc
```

Comment:
remove this line please. Its added by the script, and this file is concatenated to xcsoar-waypoints.

Reply:
Removing this line makes it impossible for SeeYou or possibly other software to read this cup file and accoring to the cup file format description you mentioned it is part of the file format.

Reply:
If this is however a xcs dedicated requirement, I'll remove the line.